### ISSUE TITLE
TypeError: _reactNativeScreens.screensEnabled is not a function

### ISSUE THREAD

**Issue by dafivius** [react-navigation/react-navigation#5015]
Hi all. Can't create bottomTabNavigator with stack navigators, getting that issue.
I'm using expo. Here is package.json
``` {

"name": "empty-project-template",
"main": "node_modules/expo/AppEntry.js",
 "private": true,
"scripts": {
    "start": "expo start",
    "android": "expo start --android",
    "ios": "expo start --ios",
    "eject": "expo eject"
  },
  "dependencies": {
    "expo": "^30.0.1",
    "native-base": "^2.8.0",
    "react": "16.3.1",
    "react-native": "https://github.com/expo/react-native/archive/sdk-30.0.0.tar.gz",
    "react-native-event-listeners": "^1.0.3",
    "react-native-gifted-chat": "^0.4.3",
    "react-navigation": "^2.16.0",
  }
}
```

Here is simple code:
```
import {
    createBottomTabNavigator,
    createStackNavigator,
} from 'react-navigation';
import React from "react";

class A extends React.Component {
    static navigationOptions = {
        tabBarLabel: 'Home!',
    };

    // etc..
}

class B extends React.Component {
    static navigationOptions = {
        tabBarLabel: 'Settings!',
    };

    // etc..
}

let HomeStack = createStackNavigator({ A });
let SettingsStack = createStackNavigator({ B });

export default BottomNavigator = createBottomTabNavigator({
    HomeStack,
    SettingsStack,
});
```
| software         | version
| ---------------- | -------
| react-navigation | 2.16.0
| node             | 9.7.1
| npm      | 6.4.1

![img_4387](https://user-images.githubusercontent.com/1564043/45824978-da4a9680-bcf9-11e8-9f27-6b7015edc233.jpg)

When i'm inserting just view not stacknavigator in route - everything works fine. Stacknavigator always fails.

What's wrong and how to defeat that bug?

**Comment by brentvatne** [comment]
potentially caused by an inconsistency between how npm and yarn resolve dependencies..... I’ll try this with your npm version to see

**Comment by jeremypagley** [comment]
@brentvatne any luck on this?

**Comment by brentvatne** [comment]
@jeremypagley - I'm looking into it right now, just use the previous minor version for now

**Comment by brentvatne** [comment]
you can do `rm -rf node_modules && rm package-lock.json && npm install` and it will resolve your issue

**Comment by brentvatne** [comment]
this bug is as a result of how expo packages new projects. when you create a new project, expo downloads a tarfile with the source code along with a node_modules directory bundled in it, so you don't need to run npm install. at the time the tarfile was created, expo's dependency on react-native-screens@^1.0.0-alpha.5 resolved to react-native-screens@1.0.0-alpha.5 (the latest version then). when you run `npm install react-navigation`, its dependency on react-native-screens is resolved to `1.0.0-alpha.11`, but expo's installed version remains locked. the dependency was also hoisted to the top level of node_modules, and since react-navigation-tabs depends on react-native-screens as a peerDependency it uses that version rather than the one within react-navigation.

so, as i mentioned above you can run `rm -rf node_modules && rm package-lock.json && npm install` to fix this. i'll re-release the expo templates so this problem doesn't arise in the future
